pom.xml

```diff
@@ -1130,6 +1133,11 @@
 					<name>imageio-ext Repository</name>
 					<url>http://maven.geo-solutions.it/</url>
 				</repository>
+				<repository>
+					<id>codice</id>
+					<name>Codice</name>
+					<url>http://artifacts.codice.org/content/groups/public/</url>
```

Comment:
http://artifacts.codice.org/content/groups/public no longer exists (It returns 404).   The dependencies from this should have been captured in the conexta-snapshots repository.  Were you having problems with resolving the Codice libraries?

Reply:
Strike that.   conexta-snapshots had the jar file in it until a few days ago (Which, at that time, artifacts.codice.org was empty).  We'll leave both repositories in there for now in hopes that it's picked up in one of them.